AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/EditorQtContainer.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Object to house all the Qt Objects used when testing and manipulating the O3DE UI
+"""
+import pyside_utils
+import editor_python_test_tools.wrapper_qt_sc_editor as wrapper_sc_editor
+import editor_python_test_tools.wrapper_qt_variable_manager as wapper_variable_manager
+
+class EditorQtContainer:
+
+    def __init__(self):
+        self.editor_main_window = pyside_utils.get_editor_main_window()
+        self.wrapper_sc_editor = None
+        self.sc_editor_main_window = None
```

Comment:
Since these are closely named to each other, lets make sure that they're documented.  Maybe in the __init__ or class level docstrings can handle this.

Reply:
made a note as to why we are doing this. we may separate these two references into two different QtPy classes in teh future. Depends what the other test scripts need